.run/secondary-watcher-pg.run.xml

```diff
@@ -0,0 +1,17 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
These are just stored IntelliJ run configs, to make things easier for others in the future.